docs/reference/deploy/master-config-reference.rst

```diff
@@ -352,7 +352,13 @@ on using Determined with Kubernetes, see the :ref:`documentation <determined-on-
 ``namespace``
 -------------
 
-The namespace where Determined will deploy Pods and ConfigMaps.
+Optional. This field has been deprecated, use ``default_namespace`` instead.
```

Comment:
Deprecated. Use ``default_namespace`` instead.

Reply:
(since it's deprecated, we should tell them that instead of telling them that it's "optional")

Reply:
done